lib/SVF-FE/SymbolTableBuilder.cpp

```diff
@@ -206,6 +209,24 @@ void SymbolTableBuilder::collectNullPtrBlackholeSyms(const Value *val)
         symInfo->blackholeSyms.insert(val);
 }
 
+void SymbolTableBuilder::collectArgInNoCallerFunction(const Value *val)
```

Comment:
merge these three functions into a `collectSpecialSym(const Value* val)` and invoked by `collectSym `